lib/committee/test/methods.rb

```diff
@@ -1,18 +1,22 @@
 module Committee::Test
   module Methods
     def assert_schema_conform
-      @schema ||= Committee::Middleware::Base.get_schema(committee_options)
-      @router ||= @schema.build_router(committee_options)
+      assert_request_schema_confirm unless old_behavior
+      assert_response_schema_confirm
+    end
 
-      v = @router.build_schema_validator(request_object)
+    def assert_request_schema_confirm
+      schema_validator.request_validate(request_object)
```

Comment:
I think we should display an error if this endpoint's schema is not defined like line 13.
(Please be careful it's different in the endpoint which does not have request schema definition and the endpoint's schema doesn't exist)

Reply:
It appeared to be returning in the request_schema_validation method.
Could you tell me a little more if you should do another check?

https://github.com/interagent/committee/blob/master/lib/committee/schema_validator/open_api_3.rb#L54
https://github.com/interagent/committee/blob/master/lib/committee/schema_validator/hyper_schema.rb#L78

Reply:
Thank you for the polite explanation!
I tried it, And added a test of request and response.
https://github.com/interagent/committee/pull/227/commits/7316598680c5fd808f1547dc112342262a5d7cd3